controller/metrics/workqueue.go

```diff
@@ -0,0 +1,123 @@
+/*
```

Comment:
I think this copyright should be removed. Code was copied but just because there are not many options of how metrics provider could be implemented.

Reply:
Done